draft-tuexen-opsawg-pcapng.xml

```diff
@@ -267,6 +268,11 @@
             capture has been made. If this appears in a file, an Interface
             Description Block is also required, before this block.</t>
 
+            <t><xref target="section_cib">Capture Information Block</xref>: it
+            defines how to store some capture information (such as location, 
+            orientation, velocity and associated errors, as well as rf frequencies 
```

Comment:
Should "rf" be capitalized (it's an initialism for "radio frequency", and I generally see it capitalized)?